action.yml

```diff
@@ -124,3 +124,26 @@ runs:
       env:
         SKIP_IMAGE_VALIDATION: ${{ inputs.skip-image-validation == 'true' && 'true' || 'false' }}
         PLATFORM: ${{ inputs.platform }}
+        
+    - id: publishbuildartifact
+      name: Publish build artifact
+      uses: docker://alpine/curl:latest
+      run: |
+        CHART_NAME=$(echo "$DESTINATION" | awk -F '[:/]' '{print $(NF-1)}')
+        CHART_VERSION=$(echo "$DESTINATION" | awk -F ':' '{print $NF}')
+        # Publish build artifact
+        response=$(curl --fail-with-body  -X 'POST' "$URL/v2/workflows/runs/artifactinfos" -H "Authorization: Bearer $API_TOKEN" -H 'Content-Type: application/json' --data-binary '{"runId":"'"$RUN_ID"'","runAttempt":'$RUN_ATTEMPT',"name":"'"$CHART_NAME"'","version":"'"$CHART_VERSION"'","url":"'"$DESTINATION"'","type":"docker"}') || command_failed=1
```

Comment:
Calling the env vars here `CHART_*` is misleading since this is about copying a container image (which may or may not be a Helm chart). How about calling the env vars `IMAGE_NAME` and `IMAGE_TAG`?

Reply:
Updated